doc/source/install-on-ubuntu.md

```diff
@@ -52,7 +52,14 @@ pip install aioredis asyncio websockets
 Then install
 [polymer](https://www.polymer-project.org/1.0/docs/tools/polymer-cli), which
 also requires [Node.js](https://nodejs.org/en/download/) and
-[Bower](http://bower.io/#install-bower).
+[Bower](http://bower.io/#install-bower).  You can run the following commands to do this: 
+
+```
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+sudo apt-get install -y nodejs
+npm install -g bower
```

Comment:
I've had trouble with this approach before. I tried following these instructions on Linux and when I hit the line `npm install -g bower` I got the error

```
npm WARN checkPermissions Missing write access to /usr/lib/node_modules
/usr/lib
└── bower@1.8.0 

npm ERR! Linux 4.4.0-64-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "bower"
npm ERR! node v6.10.0
npm ERR! npm  v3.10.10
npm ERR! path /usr/lib/node_modules
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall access

npm ERR! Error: EACCES: permission denied, access '/usr/lib/node_modules'
npm ERR!     at Error (native)
npm ERR!  { Error: EACCES: permission denied, access '/usr/lib/node_modules'
npm ERR!     at Error (native)
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/lib/node_modules' }
npm ERR! 
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/npm-debug.log
```

I think I had to solve this in the past by just getting the node binaries from https://nodejs.org/en/download/ or something like that.